
string name #Name of the location 

---

bool Status #Execution Status